packages/connect-wallet-modal/src/components/WalletsModal/WalletsModal.tsx

```diff
@@ -147,7 +64,46 @@ export function WalletsModal(props: WalletsModalProps) {
         </Modal>
       );
     }
-  }
 
-  return null;
+    case 'ledger':
+      return (
+        <LedgerModal
+          {...passedDownProps} // the props are overridden here on purpose
+          open
+          onClose={handleCloseSuccess}
+          onBack={handleCloseReject}
+          onExited={handleExit}
+          metrics={metrics}
+        />
+      );
+
+    case 'eager':
+      return (
+        <EagerConnectModal
+          tryConnecting={currentModal.props.tryConnection}
```

Comment:
`tryConnecting` sounds wrong, `tryConnection` is better. Also, in one place you have `tryConnecting` and `tryConnection` in another, it may be confusing.